doc/topics/installation/osx.rst

```diff
@@ -65,16 +137,22 @@ When only using the macOS system's pip, install this way:
 
 Salt-Master Customizations
 ==========================
+
 .. note::
-    Salt master on macOS is not tested or supported by SaltStack. See `SaltStack Platform Support <https://saltstack.com/product-support-lifecycle/>`_ for more information.
 
-To run salt-master on macOS, sudo add this configuration option to the /etc/salt/master file:
+    Salt master on macOS is not tested or supported by SaltStack. See
+    `SaltStack Platform Support <https://saltstack.com/product-support-lifecycle/>`_
+    for more information.
+
+To run salt-master on macOS, sudo add this configuration option to the
+``/etc/salt/master`` file:
 
 .. code-block:: bash
 
     max_open_files: 8192
 
-On versions previous to macOS 10.10 (Yosemite), increase the root user maxfiles limit:
+On versions previous to macOS 10.10 (Yosemite), increase the root user maxfiles
```

Comment:
We no longer support macOS10.10.  Can you remove this section?

Reply:
Done